ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -104,6 +104,27 @@ if get_option('enable-tensorflow-lite')
     endif
   endif
 
+  tflite_compile_args = []
+
+  if cxx.has_type('kTfLiteInt8', prefix : '#include <tensorflow/contrib/lite/model.h>')
```

Comment:
Is there any reason that you picked `model.h` as a prefix?
+ for version 1.15 & 1.13, the path of `model.h` is `<tensorflow/lite/model.h>`

Reply:
@helloahn This is because you include that header in the tensor filter extension for tflite:
https://github.com/nnsuite/nnstreamer/blob/c000f9e3d93c213291f531af7e6c98fff1e230b8/ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc#L35

Reply:
> As you already knew, we have provided include/tensorflow/contrib/* for backward compatibility.

Oh, I forgot about it. Thanks for reminding me.